diff --git a/soph/train.py b/soph/train.py
index 437aff0..9d9de67 100644
--- a/soph/train.py
+++ b/soph/train.py
@@ -653,7 +653,7 @@ def train(*,out_dir='out', eval_interval=2000, log_interval=100, eval_iters=200,
                 Ts = np.array(log_df['tokens']) # (iter,)
                 L = current_logs['train_loss']
                 dLs = Ls - L
-                auc = np.trapz(dLs, Ts)
+                auc = np.trapezoid(dLs, Ts)
                 current_logs['K_auc'] = auc
                 log_df['K_auc'].append(auc)
             if wandb_log:
